tests/drivers/dma/loop_transfer/src/test_dma_loop.c

```diff
@@ -481,9 +464,13 @@ static int test_loop_repeated_start_stop(const struct device *dma)
 	TC_PRINT("Each RX buffer should contain the full TX buffer string.\n");
 
 	for (int i = 0; i < TRANSFER_LOOPS; i++) {
-		TC_PRINT("RX data Loop %d: %s\n", i, rx_data[i]);
-		if (strncmp(tx_data, rx_data[i], sizeof(rx_data[i])) != 0) {
-			return TC_FAIL;
+		TC_PRINT("RX data Loop %d\n", i);
+		for (int j = 0; j < XFER_SIZE; j++) {
```

Comment:
If memcmp is available that should be used instead

Reply:
Done.